src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -468,6 +517,64 @@ public override async Task RewindInstanceAsync(
         }
     }
 
+    /// <inheritdoc/>
+    public override async IAsyncEnumerable<HistoryEvent> StreamInstanceHistoryAsync(
+        string instanceId,
+        string? executionId = null,
+        CancellationToken cancellation = default)
+    {
+        Check.NotNullOrEmpty(instanceId);
+        Check.NotEntity(this.options.EnableEntitySupport, instanceId);
+
+        P.StreamInstanceHistoryRequest request = new()
+        {
+            InstanceId = instanceId,
+            ExecutionId = executionId,
+            ForWorkItemProcessing = false,
+        };
+
+        using AsyncServerStreamingCall<P.HistoryChunk> streamResponse =
+            this.sidecarClient.StreamInstanceHistory(request, cancellationToken: cancellation);
+
+        IAsyncStreamReader<P.HistoryChunk> responseStream = streamResponse.ResponseStream;
+
+        // Create conversion state to maintain orchestration instance across events
+        // This is required for entity-related events (EntityOperationCalled, EntityLockRequested, etc.)
+        // which need the parent orchestration instance information from ExecutionStartedEvent
+        Microsoft.DurableTask.ProtoUtils.EntityConversionState conversionState = new(insertMissingEntityUnlocks: false);

```

Comment:
Are entities exported in the same way as an orchestration's history?

Reply:
no entities not supported, they wont have history events to export